homeassistant/components/vera/strings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "config": {
+    "title": "Vera",
+    "abort": {
+      "already_setup": "A controller is already configured.",
+      "cannot_connect": "Could not connect to controller with url {base_url}"
+    },
+    "step": {
+      "user": {
+        "title": "Setup Vera controller",
+        "description": "Provide a Vera controller url below. It should look like this: http://192.168.1.161:3480.",
+        "data": {
+          "vera_controller_url": "Controller URL",
+          "lights": "Vera switch device ids to treat as lights in Home Assistant.",
+          "exclude": "Vera device ids to exclude from Home Assistant."
+        }
+      }
+    }
+  },
+  "options": {
+    "abort": {
+      "file_based_config": "Configuration for this Vera controller is handled in configuration.yml. Please make changes there."
+    },
+    "step": {
+      "init": {
+        "title": "Vera controller options",
+        "description": "See the vera documentation for details on optional parameters: https://www.home-assistant.io/integrations/vera/. Note: Any changes here will need a restart to the home assistant server.",
+        "data": {
+          "lights": "Vera switch device ids to treat as lights in Home Assistant.",
+          "exclude": "Vera device ids to exclude from Home Assistant."
+        }
+      },
+      "from_config": {
+        "title": "UI config not supported",
+        "description": "Controller was configured using configuration.yml. Config changes should be made in the config file."
```

Comment:
This is weird. Either we support GUI options all the way, or we should remove it.